content/blog/2020-11-30-super-optimization/index.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Super Optimization"
+[extra]
+bio = """
+  Socrates Wong is a secound year PhD student at Cornell who intreset lies in Computer Archicture.
+"""
+latex = true
+[[extra.authors]]
+name = "Socrates Wong"
++++
+
+## Background
+Most complier optimization (peephole, dead code elimination, constant propagation, loop invariant code motion, common sub expression elimination) focus on transforming the program in a systematic manner, and the step taken by the optimizer are backed by formal rigorous proofs that are both sound and complete.  Superoptimizer is a new radical way of preforming optimization.  Instead of taking the traditional approach optimizers of transforming the input program, superoptimizer converts the problem in to a search space and attempts to find the shortest program that is able to provide the equivalent output.
+The paper we are discussing this week is ["Superoptimizer"][superopt] – A look at the Smallest Program” (Henry), this paper explains the purpose of superoptimizer, which converts an optimizing problem to a search space, how to reduce the search space and comparison with other common optimization techniques that are common at the publication of the paper. 
+
+
+## Super Optimization
```

Comment:
Please pick either "superoptimization" or "Super Optimization" and stick with it consistently throughout the blog post. (Probably the former, not capitalized, because that's what the paper uses.)